spec/jobs/broadcast_compatible_targets_job_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe BroadcastCompatibleTargetsJob, type: :job do
+  include ActiveJob::TestHelper
+
+  subject { described_class.perform_now(target) }
+
+  let(:lat) { Faker::Address.latitude }
+  let(:lng) { Faker::Address.longitude }
+  let(:serialized_target) { ActiveModelSerializers::SerializableResource.new(target).as_json }
+  let(:target) { create :target, topic: topic, lat: lat, lng: lng, area_length: 1500 }
+  let(:topic) { create :topic }
+
+  context 'new target with compatible targets' do
+    let(:compatible_target) do
+      create :target, topic: topic, area_length: 2000, lat: lat + 0.001,
+                      lng: lng + 0.001
+    end
+
+    before do
+      compatible_target
+    end
+
+    it 'should broadcast to the users with compatible targets' do
```

Comment:
Don't use `should` in expectation description. 
`it 'broadcasts to the users with compatible targets'`